consumers/messages.go

```diff
@@ -59,7 +82,10 @@ func handle(t transformers.Transformer, c Consumer) handleFunc {
 				return err
 			}
 		}
-		return c.Consume(m)
+
+		errs := ac.Errors()
+		ac.ConsumeAsync(m)
+		return <-errs
```

Comment:
There is no point in waiting for errors - this results in a blocking operation, which is counter-productive. It also makes the writer required to send `nil` errors in case of a successful operation. We handle errors elsewhere, so simply return nil here. Subscriber Handler is successful, and Consumer might fail, but those errors are handled in a separate routine.